client/blocks/reader-post-card/style.scss

```diff
@@ -256,11 +256,11 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 }
 
 .reader-post-card__tag {
+	flex: 1 0 0;
```

Comment:
absolutely incredible how many places this has bitten us
I wonder if other bubbles of a8c have run into the same issue and/or if we could prevent it for them?